packages/dnb-eufemia/src/components/copy-on-click/CopyOnClickDocs.ts

```diff
@@ -16,6 +16,11 @@ export const CopyOnClickProperties: PropertiesTableProps = {
     type: 'React.Node',
     status: 'optional',
   },
+  copiedFeedbackMessage: {
```

Comment:
```suggestion
  feedbackMessage: {
```

Do we need to include `copied`? I think it's still very understandable with such a shorter name.

Reply:
But maybe its more explicit? What about using the word "Tooltip"? Something like `tooltipContent`?

Reply:
Changed to `tooltipContent` now 🙏